src/components/essences/essences-toolbar.tsx

```
@@ -1,97 +1,145 @@
+            </div>
+          )}
+          {raritiesPending ? (
+            <span className="loading loading-xs" />
+          ) : (
+            <ChevronDown size="1.2em" />
+          )}
+        </summary>
+        <ul
+          className="menu dropdown-content card z-1 mt-2 w-full shadow-2xl sm:min-w-48"
+          // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex -- needed by daisyUI to prevent focus loss
+          tabIndex={0}
```

Biome (2.1.2)

[error] The HTML element ul is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)